### DISCUSSION TITLE
Please add vertical right slider

### DISCUSSION THREAD

**Discussion by tomekjordan** [ChristopherKlay/StadiaEnhanced#56] [Ideas & Requests]
Stadia store UX is horrible, they even disabled default vertical scroller. Thus I cannot buy games on desktop.
The only way is to scroll down on mobile - and only mobile/ipad support TOUCH MODE.

Slider should be enabled by default on desktop - please fix this annoying issue. I cannot believe Google UX is such bad,
it totally sucks

thanks for help

**Comment by ChristopherKlay** [comment]
Just added the option to the dev version.

Will be included with `v1.8.0` that should drop in a day or two (:
